src/mito2/src/engine.rs

```diff
@@ -415,14 +415,20 @@ impl MitoEngine {
 
     /// Lists all SSTs from the manifest of all regions in the engine.
     pub fn all_ssts_from_manifest(&self) -> impl Iterator<Item = ManifestSstEntry> + use<'_> {
+        let node_id = self.inner.workers.file_ref_manager().node_id();
```

Comment:
Do we need node_id? We can use the region id to find the node later.

Reply:
This field is primarily used in distributed mode to distinguish the source of results. Having this field does no harm and serves as additional verification when multiple confirmations are needed.